continuous_integration/envs/39-dev.yaml

```diff
@@ -15,10 +15,11 @@ dependencies:
   - pytest
   - pytest-cov
   - hilbertcurve
+  - pymorton
```

Comment:
Remove this one.

Reply:
> Remove this one.

ah, I just forgot to remove it as a conda dependency :facepalm:

updated in 72bb05d